lib/Table/TableAutomaticStylesStratum.ts

```diff
@@ -283,7 +281,10 @@ export class ColorStyleLegend extends LoadableStratum(LegendTraits) {
     ];
   }
 
-  private _formatValue(value: number, format: JsonObject | undefined): string {
+  private _formatValue(
+    value: number,
+    format: Intl.NumberFormatOptions | JsonObject | undefined
+  ): string {
```

Comment:
Just a small refactor. Earlier in the PR I made a change to this, then rolled it back and left the refactor.